operator/v1/types_csi_cluster_driver.go

```diff
@@ -105,10 +105,13 @@ type ClusterCSIDriverSpec struct {
 }
 
 // CSIDriverType indicates type of CSI driver being configured.
-// +kubebuilder:validation:Enum="";vSphere
+// +kubebuilder:validation:Enum=AWS;Azure;GCP;vSphere
```

Comment:
Isn't `""` still valid?

Reply:
Yeah - we should still keep `""` until we can fix the defaulting for upgraded clusters.

Reply:
Fixed.